notebooks/Deploy summarize.ipynb

```diff
@@ -82,12 +82,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Mapping for Model Packages (initially only us-east-1 and eu-west-1 is supported)\n",
+    "cohere_package = \"cohere-summarize-v0-1\"\n",
```

Comment:
When published, the package name is something like cohere-gpt-xlarge-v1-2-4d938caa0259377e94c4eb5bf6bc365a


You don't need to create a package per region, it will be available in all region but we have to wait marketplace listing.